src/main/kotlin/no/nav/tilleggsstonader/sak/vedtak/boutgifter/beregning/BoutgifterBeregningServiceFeilmeldingUtil.kt

```diff
@@ -37,7 +37,15 @@ object BoutgifterBeregningServiceFeilmeldingUtil {
 
     private fun UtgiftBeregningBoutgifter.tilOverlappendePerioderPunktliste(utbetalingsperioder: List<UtbetalingPeriode>): String =
         finnOverlappendeUtbetalingsperioder(utbetalingsperioder)
-            .joinToString(" og") { " ${it.formatertPeriodeNorskFormat()}" }
+            .mapNotNull { utbetalingPeriode ->
+                val overlapFom = maxOf(this.fom, utbetalingPeriode.fom)
+                val overlapTom = minOf(this.tom, utbetalingPeriode.tom)
+                if (overlapFom <= overlapTom) {
+                    UtgiftBeregningBoutgifter(overlapFom, overlapTom, utgift = this.utgift, this.skalFåDekketFaktiskeUtgifter)
+                } else {
+                    null
+                }
+            }.joinToString(" og") { " ${it.formatertPeriodeNorskFormat()}" }
```

Comment:
Småting, men følte det ble litt vanskelig å lese med spaces og template.

Her går det å fjerne template string, å bruke `it.formatertPeriodeNorskFormat()` direkte
```suggestion
            }.joinToString(" og ") { it.formatertPeriodeNorskFormat() }
```

Man må då også oppdatere `append("${formatertPeriodeNorskFormat()} må splittes til${tilOverlappendePerioderPunktliste(utbetalingsperioder)}")` 
 med en space mellom `til` og `$`

Reply:
Oppdatert -  .joinToString("\n") { " - ${it.formatertPeriodeNorskFormat()}" 

lagt til ekstra "-" også . Er det ok nå?